content/docs/pipelines/sdk/dsl-recursion.md

```diff
@@ -0,0 +1,91 @@
++++
+title = "DSL Recursion"
+description = "Author a recursive function in DSL."
+weight = 7
++++
+
+This page describes how to write recursive functions in DSL.
+
+## Motivation
+Recursion is a feature that is supported by almost all languages to express complex semantics in a succinct way. 
+In machine learning workflows, recursions are especially important to enable features such as multiple rounds of training, 
+iterative model analysis, and hypertuning. The recursion support also covers the loop feature since it enables the same code 
+block to be executed and exited based on dynamic conditions.
+
+## How to write a recursive function?
+
+### Decorator
+
+Decorate the recursive function with [kfp.dsl.graph_component](https://github.com/kubeflow/pipelines/blob/master/sdk/python/kfp/dsl/_component.py)
+as illustrated below. No arguments to the decorator is required.
```

Comment:
Change the last sentence to active voice, as it's clearer. Also, it avoids the awkwardness of needing to change "is" to "are" to agree with the plural subject "arguments".

The decorator does not require any arguments.

Reply:
SG. Done.